sqs_client_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build sqsint
 // +build sqsint
```

Comment:
You can remove these now

Reply:
I added this back, as CI shouldn't run these integration tests. 

Can you suggest a better way to ignore these tests?

Reply:
The new tags ( `//go:build sqsint` ) replace the old ones ( `//+build sqsint` ) in go 1.18 (https://tip.golang.org/doc/go1.18#go-build-lines).

You still need the tags, but you don't need both the new and the old.

Reply:
Got it, thanks 👍